script/build.js

```diff
@@ -80,6 +80,7 @@ smith.destination(`../build/${options.buildtype}`);
 const ignoreList = ['memorial-benefits/*'];
 if (options.buildtype === 'production') {
   ignoreList.push('rx/*');
+  ignoreList.push('facility-locator/*');
```

Comment:
@manfromtexas this should be `/facilities`

https://github.com/department-of-veterans-affairs/vets-website/blob/719cff51a7a296cfb46ba20b7f90aa644702846f/src/js/facility-locator/routes.jsx#L6

`facility-locator` is the old version which we don't want to take offline until this new one is complete